playground/scenarios/demo-app/metadata.json

```diff
@@ -6,7 +6,9 @@
     {
       "blueprint_name": "policies/latency-aimd-concurrency-limiting",
       "policy_name": "service1-demo-app",
-      "values_file": "policies/service1-demo-app.yaml"
+      "values_file": "policies/service1-demo-app.yaml",
```

Comment:
What about signals dashboard? How is that generated?

Reply:
Signals dashboard is still coming from `lib/apps` as we don't have it in scenarios. So basically it will always be there when we run playground.

Reply:
Okay, the panels in it are filtered by policy name. We should either make policy one of the variables so that it works for all policies or generate it as part of scenario

Reply:
Okay, I'll check it tomorrow and most probably will generate as part of the scenarios.

Reply:
Handled this as part of https://github.com/fluxninja/aperture/pull/1500